bundles/org.openhab.binding.somfytahoma/README.md

```diff
@@ -58,7 +59,7 @@ Please see the example below.
 | gateway       | status  | status of your Tahoma gateway |
 | gate          | gate_command | used for controlling your gate (open, close, stop, pedestrian) |
 | gate          | gate_state | get state of your gate |
-| roller shutter, screen, venetian blind, garage door, awning, window, pergola | control |  device controller which reacts to commands UP/DOWN/STOP + closure 0-100 |
+| roller shutter, screen, venetian blind, garage door, awning, window, pergola, curtain | control |  device controller which reacts to commands UP/DOWN/STOP + closure 0-100 |
```

Comment:
Only nitpicking: Can you apply the markdown table formatter (see https://www.tablesgenerator.com/markdown_tables)?